pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -51,24 +30,18 @@ stdenv.mkDerivation rec {
     pkgconfig perl perlXMLParser libXft libpng zlib popt boehmgc
     libxml2 libxslt glib gtkmm2 glibmm libsigcxx lcms boost gettext
     makeWrapper intltool gsl poppler imagemagick libwpg librevenge
-    libvisio libcdr libexif automake114x autoconf
-  ] ++ stdenv.lib.optional boxMakerPlugin unzip;
+    libvisio libcdr libexif automake114x cmake
+  ];
 
   enableParallelBuilding = true;
-  doCheck = true;
 
-  postInstall = ''
-    ${if boxMakerPlugin then "
-      mkdir -p $out/share/inkscape/extensions/
-      # boxmaker packaged version 0.91 in a directory called 0.85 ?!??
-      unzip ${boxmaker};
-      cp boxmake-upd-0.85/* $out/share/inkscape/extensions/
-      rm -Rf boxmake-upd-0.85
-      "
-    else 
-      ""
-    }
+  configurePhase = ''
```

Comment:
Have you tried not specifying `configurePhase`? It looks like what would happen by default when you just add `cmake` to `buildInputs` (because of a `setup-hook` in `cmake`)